uploader.json

```diff
@@ -2562,6 +2562,11 @@
             "file": "/var/log/rhn/rhn_taskomatic_daemon.log",
             "pattern": [],
             "symbolic_name": "rhn_taskomatic_daemon_log"
+        },
+	   {
+            "file": "/etc/rhosp-release",
+            "pattern": [],
+            "symbolic_name": "rhosp_release"
```

Comment:
@psachin please fix the indentation above and in the changes for `uploader.v2.json`.

Reply:
Done